File: temp-ml-button-fix.php
<?php
$root = __DIR__ . '/resources/views/admin/converted-leads';
$pattern = '/(?P<indent>^\s*)<a href="{{ route\\(\'admin\\.digital-marketing-mentor-converted-leads\\.index\'\) }}" class="btn btn-outline-primary(?: active)?">\s*<i class="ti ti-user-star"><\\/i> Digital Marketing Mentor List\s*<\\/a>/m';
$insertTpl = "<a href=\\\"{{ route('admin.machine-learning-mentor-converted-leads.index') }}\\\" class=\\\"btn btn-outline-primary\\\">\n%indent%    <i class=\\\"ti ti-user-star\\\"></i> Machine Learning Mentor List\n%indent%</a>";
$updated = [];
$dir = new RecursiveDirectoryIterator($root);
$it = new RecursiveIteratorIterator($dir);
foreach ($it as $file) {
    if (!$file->isFile() || $file->getExtension() !== 'php') continue;
    $path = $file->getPathname();
    $contents = file_get_contents($path);
    if (strpos($contents, 'machine-learning-mentor-converted-leads') !== false || strpos($contents, 'digital-marketing-mentor-converted-leads') === false) {
        continue;
    }
    $new = preg_replace_callback($pattern, function($m) use ($insertTpl) {
        $indent = $m['indent'];
        $insert = str_replace('%indent%', $indent, $insertTpl);
        return $m[0] . "\n" . $indent . $insert;
    }, $contents, -1, $count);
    if ($count > 0) {
        file_put_contents($path, $new);
        $updated[] = [$path, $count];
    }
}
foreach ($updated as [$p, $c]) {
    echo "updated {$p} ({$c})\n";
}
echo "total updated: " . count($updated) . "\n";
?>

File: temp-ml-button-fix.py
from pathlib import Path
import re

root = Path('resources/views/admin/converted-leads')
pattern = re.compile(r'(?P<indent>\s*)<a href="{{ route\(\'admin\.digital-marketing-mentor-converted-leads\.index\'\) }}" class="btn btn-outline-primary(?P<active>\s+active)?">\s*<i class="ti ti-user-star"></i> Digital Marketing Mentor List\s*</a>', re.MULTILINE)
insert_template = "{indent}<a href=\"{{ route('admin.machine-learning-mentor-converted-leads.index') }}\" class=\"btn btn-outline-primary\">\n{indent}    <i class=\"ti ti-user-star\"></i> Machine Learning Mentor List\n{indent}</a>"
updated = []
for path in root.rglob('*.blade.php'):
    text = path.read_text()
    if 'machine-learning-mentor-converted-leads' in text or 'digital-marketing-mentor-converted-leads' not in text:
        continue
    def repl(match):
        indent = match.group('indent')
        return match.group(0) + "\n" + insert_template.format(indent=indent)
    new_text, count = pattern.subn(repl, text)
    if count:
        path.write_text(new_text)
        updated.append((path, count))
for p, c in updated:
    print(f"updated {p} ({c})")
print(f"total updated: {len(updated)}")
